integration-tests/zombienet/tests/smoke.rs

```diff
@@ -0,0 +1,29 @@
+use std::time::Instant;
+use subxt::ext::futures::StreamExt;
+use zombienet_sdk_tests::{environment::get_spawn_fn, small_network, wait_subxt_client};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn smoke() {
+	tracing_subscriber::fmt::init();
+
+	// config and env
+	let spawn_fn = get_spawn_fn();
+	let config = small_network().unwrap();
+
+	// spawn
+	let now = Instant::now();
+	let network = spawn_fn(config).await.unwrap();
+	let elapsed = now.elapsed();
+	log::info!("🚀🚀🚀🚀 network deployed in {:.2?}", elapsed);
+
+	let alice = network.get_node("alice").unwrap();
```

Comment:
Where can I find the names of the nodes? How do I know if a node belongs to a parachain or the relay chain?

Reply:
The `config` came from [small_network](https://github.com/polkadot-fellows/runtimes/pull/307/files#diff-d645fab731f2bf1bbd7e52d36bcf30a59649d7bda90e16a7ddd0719da1982d2dR11) and I placed there for reuse in other tests, but could be better to each test have it's own config in the same file to easily find the nodes/paras. I can change if you are agree.

Thx!

Reply:
Ahh oks, in the `sdk` you have to set the `name` of the node with a like like this

```rust
.with_node(|node| node.with_name("alice"))
```
but we can make more explicit as `const` to be easily find with a quick look. wdyt?

Reply:
```
.with_node(|node| node.with_name("alice"))
```

But you don't do this here anywhere in the code? Or am I blind?

Reply:
Yes, we do as part of the `small_network` fn here https://github.com/polkadot-fellows/runtimes/pull/307/files#diff-d645fab731f2bf1bbd7e52d36bcf30a59649d7bda90e16a7ddd0719da1982d2dR27-R28

I changed to use `consts`, e.g:

```rust
// Relaychain nodes
const ALICE: &str = "alice";
const BOB: &str = "bob";

... 

.with_node(|node| node.with_name(ALICE))
.with_node(|node| node.with_name(BOB))
```

Thx!

Reply:
Fuck I'm blind :facepalm: :facepalm:

Reply:
Thanks for the feedback! 🙌 

Yes, the _naming_ could be confusing. Both method `with_node` and `with_collator` return a `NodeConfigBuilder` that by default is _validator_ (this is also confusing because for nodes of a parachain means that we pass the flag `--collator`). 
Maybe is better to just have `with_node` with the default `_role_` set to `ROLE::AUTHORITY` and you can change to `full` with `node.role(ROLE::FULL)`

So, this way is more clear and use the same wording you see in the logs of the client. wdyt?

Thx!

Reply:
> So, this way is more clear and use the same wording you see in the logs of the client. wdyt?

Or just call them `with_validator`, `with_node` and `with_collator`. Probably the least confusing one. Otherwise I would go with `with_node` and require that you make it a validator/collator (but the argument against this being that you almost never need a normal full node in the tests)

Reply:
Yes, I think is more explicit and will cause less confusion have those 3 methods 👍.
Thanks again for the feedback!